### ISSUE TITLE
Ultimate Romy r6 PCB question

### ISSUE THREAD

**Issue by christopherpm** [reinauer/amiga-romy#6]
Hi, the holes under the TSOP pins 13 through 24 - are they vias?

**Comment by reinauer** [comment]
Yes, they connect the two footprints for the 22V10s so you can choose which one to stuff depending on whether you build a piggy back version (A4000) or a PLCC plug version (A3000). I might reroute the PCB to get rid of those vias at some point, but I have another experiment I want to do before redoing this. The design does work in both machine types though, in all tested board revisions
